test/capwap_loc_provider_SUITE.erl

```diff
@@ -0,0 +1,176 @@
+%% Copyright (C) 2018, Travelping GmbH <info@travelping.com>
```

Comment:
If you start update copyright -  why here present only 2018?

Reply:
I'll update the tests too.